Torrent based DataSource.

jaideep-seth opened the pull request:
#409 
- Added Informational for torrent_file_name.

[Sample_Output](https://drive.google.com/open?id=1ysMqvnf1IdHIHMyvaYYmy6Bb4ivqYvuX)

mwatts15 commented on PyOpenWorm/context_store.py:
```diff
@@ -151,7 +151,15 @@ def triples(self, pattern, context=None):
         for t in self.__store.triples(pattern, context):
             contexts = set(getattr(c, 'identifier', c) for c in t[1])
-            inter = self.__context_transitive_imports & contexts
+            if self.__context_transitive_imports:
```
Actually, should be a one line change. From your current `dev`, you could do
```
git branch my-old-dev
git reset --hard origin/dev
git cherry-pick 4cf311a554032b8e66d1d622c6ec07aacf01668c
```
and start from there.

jaideep-seth replied:
fatal: bad object 4cf311a554032b8e66d1d622c6ec07aacf01668c is what it says @mwatts15 . I reinstalled and am now making another PR...

jaideep-seth commented:
@mwatts15  i reinstalled PyOpenWorm dev branch, and added an Informational for 'torrent_file_name'.  Please let me know if this is alright. I added an image of the output as a 'Sample_Onput' in the comment above.
